package.json

```diff
@@ -47,5 +47,8 @@
     "semver": "^7.3.7",
     "tsx": "^3.8.0"
   },
+  "dependencies": {
```

Comment:
remove

Reply:
are you just pointing out the file dependency? bc analytics is going to need to be bundled with the library is it not?